### ISSUE TITLE
Customiser - Footer align

### ISSUE THREAD

**Issue by dzamelt** [dflippo/UU2014#52]
Dear Mr D Flippo,

I have couple of questions/issues using your fantastic UU2014 theme, will there be a 2015 version...

I just noticed today that the customiser panel is not functioning or showing anything. The cause is not obvious but I did edit the ... section to remove author/time and date from posts by inserting // in front of the relevant script. I wouldn't have thought that could put the customiser out....

Then I was basically wondering if I could edit the footer in such a way as to centre or left justify the content, i.e. the meta / log in - or copyright section.... I think I will work this out but just wondered if you had a quick answer or script of could confirm if footer php needs to be edited etc...

Many thanks

**Comment by dflippo** [comment]
First, I hope you created a child theme for these changes?  If not, you will loose all your changes the next time you update the theme (and you should run the updates for the theme).  To help people, I created a starter child theme which you can download from https://github.com/dflippo/uu2014-child .  By using your own child theme, you can switch WordPress between the parent and child theme to identify if your child theme has broken something.  I recommend downloading any changes you have made so you don't loose them, delete the parent theme, install the parent theme through WordPress, and then upload your new child theme.  I am hoping that will clear up the empty customizer and let you make the changes you want.

You should not need to modify anything to hide the by-lines that show the author or date because that is a feature of the theme (it is in the customizer that isn't working for you).  Once the customizer is working you will have a section in there to display or hide features and these are things you can hide.

The footer has a widget area that allows you to add text widgets with whatever HTML, CSS, or JavaScript you want to add to the footer.  The line at the very bottom is intended to be static but you can change that by creating your own footer.php in your child theme.  I suggest trying the widget approach first to see if it gives you want you are looking for.  To change the alignment of the text, in the widget, you will want to add some CSS to the HTML you add.

I think I've answered your questions but let me know if you have a follow-up question.

**Comment by dzamelt** [comment]
Thanks a lot for the support, no didn't know anything about a child theme but I saved copies of the original script before making changes - hence i will try undoing the changes i made before reinstalling/updating. Then I will get familiar using the child theme. Many thanks sir.

**Comment by dzamelt** [comment]
Dear Dan,

I'm trying to clear my left sidebar. I've removed all the sidebar widgets etc, but it displays the navigation menu, recent post menu, archive menu and category menu. I have checked all the other settings I can think of that might give such a command but cannot see any obvious problem. When I insert a widget into the sidebar it appears underneath the menus noted. Any idea or advice on this Dan?

Note: The customizer works okay when all the plugins are deactivated. I have identified one plugin that gives issue when others are active but I'm not sure which they are yet but that plugin has been okay most of the time prior. When most of the plugins are active there is a point where the customiser continues to function except for when closing it via the x tab to which it fails to respond.

Cheers

Date: Tue, 24 Mar 2015 17:54:56 -0700

**Comment by dzamelt** [comment]
Dear Dan,

I'm trying to clear my left sidebar. I've removed all the sidebar widgets etc, but it displays the navigation menu, recent post menu, archive menu and category menu. I have checked all the other settings I can think of that might give such a command but cannot see any obvious problem. When I insert a widget into the sidebar it appears underneath the menus noted. Any idea or advice on this Dan?

Note: The customizer works okay when all the plugins are deactivated. I have identified one plugin that gives issue when others are active but I'm not sure which they are yet but that plugin has been okay most of the time prior. When most of the plugins are active there is a point where the customiser continues to function except for when closing it via the x tab to which it fails to respond.

Cheers

**Comment by dzamelt** [comment]
Okay, the problem plugin is "type case" so far as preventing the x tab to close the customiser - I think most of us can cope without the x tab closure! Can always deactivate etc accordingly. The other plugin that causes the customiser not the function is called 'geo-counter' a spinning globe that can look good in the sidebar etc. But when removed from the widget area the customiser works okay. So this faults can be worked around.

The main problem left is the menus sticking to the sidebar - I've no idea what I've done there.

Cheers

**Comment by IgnitionMetalArtDesign** [comment]
This answer Dan Flippo gave on GitHub (issue#48) and it works nicely…BTW
you can search UU2014 on GitHub and often find the solution,,,,



If you don't have any widgets in the left sidebar, the theme will display
some default content. You can avoid this by adding an empty text widget to
the area. If you want to hide the left navigation menu, you should create
an empty menu and then assign the empty menu to the sidebar navigation (you
see this relationship in Menus in the second tab).

Alternately, if you don't want the sidebar to appear at all and want
content to fill the entire width, you can do that by hiding the sidebar in
the Display and Hide Features section of the Customizer. Good luck.



Marty





*From:* dzamelt [mailto:notifications@github.com]
*Sent:* Friday, March 27, 2015 3:41 PM
*To:* dflippo/UU2014
*Subject:* Re: [UU2014] Customiser - Footer align (#52)



Okay, the problem plugin is "type case" so far as preventing the x tab to
close the customiser - I think most of us can cope without the x tab
closure! Can always deactivate etc accordingly. The other plugin that
causes the customiser not the function is called 'geo-counter' a spinning
globe that can look good in the sidebar etc. But when removed from the
widget area the customiser works okay. So this faults can be worked around.

The main problem left is the menus sticking to the sidebar - I've no idea
what I've done there.

Cheers

—
Reply to this email directly or view it on GitHub
<https://github.com/dflippo/UU2014/issues/52#issuecomment-87110744>.

**Comment by dzamelt** [comment]
Thank you Marty, thats useful to know. I think my problem was slightly different in that even when widgets were added to the sidebar it nevertheless remained full of menus etc, as if no widgets were added maybe. However, here's what seems to have fixed it. On the menu editor on the categories and pages section I selected all, then unselected and saved. That cleared them from the sidebar.

I think this is an awesome theme - I have recently linked up with my local Unitarians - and told the minister all about it! http://www.interfaithveganalliance.org/breaking-bread-homeless/

One more thing, I know there is a way to have the header images alternating within each page... I will look this up later, but if anyone has a quick remedy I will be grateful.

Many thanks!

**Comment by IgnitionMetalArtDesign** [comment]
There is a setting for that…..randomize suggested headers… ….see attached
picture



Go to customizer/header images/   then go down to bottom of that section



I just use a slider that is capable of header image function
instead….SOLILOQUY



Works for me



Thanks

Marty



*From:* dzamelt [mailto:notifications@github.com]
*Sent:* Friday, March 27, 2015 4:57 PM
*To:* dflippo/UU2014
*Cc:* Marty Slusser
*Subject:* Re: [UU2014] Customiser - Footer align (#52)



Thank you Marty, thats useful to know. I think my problem was slightly
different in that even when widgets were added to the sidebar it
nevertheless remained full of menus etc, as if no widgets were added maybe.
However, here's what seems to have fixed it. On the menu editor on the
categories and pages section I selected all, then unselected and saved.
That cleared them from the sidebar.

I think this is an awesome theme - I have recently linked up with my local
Unitarians - and told the minister all about it!
http://www.interfaithveganalliance.org/breaking-bread-homeless/

One more thing, I know there is a way to have the header images alternating
within each page... I will look this up later, but if anyone has a quick
remedy I will be grateful.

Many thanks!

—
Reply to this email directly or view it on GitHub
<https://github.com/dflippo/UU2014/issues/52#issuecomment-87127054>.
